aws-xray-recorder-sdk-core/src/main/resources/META-INF/native-image/com.amazonaws/aws-xray-recorder-sdk-core/reflect-config.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "name": "com.amazonaws.xray.AWSXRay",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.manifest.SamplingRuleManifest",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.rule.SamplingRule",
+    "allPublicMethods": true,
+    "allPublicConstructors": true,
+    "allDeclaredFields": true,
+    "allDeclaredMethods": true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name": "com.amazonaws.xray.strategy.sampling.reservoir.Reservoir",
```

Comment:
I have the same question here that @anuraaga had in the original issue, to quote him:

> My understanding is graal would only need these for code that is accessed through reflection. I understand many of the classes are because Jackson accesses the fields reflectively, but there seem to be some like "name": "com.amazonaws.xray.strategy.sampling.reservoir.Reservoir", and friends that don't. Do you know if these should be included?

Reply:
Just check I understand the question. Do you believe that this is never accessed via reflection? and isn't required?

Reply:
Yes that is my understanding, but I could be wrong. I guess the author of these configs probably listed all these classes based on experimentation. If that's the case, then it's fine, just. interesting that some part of this logic (probably via Jackson) ends up accessing this class reflectively.

Reply:
I tested the config with this project https://github.com/aws-samples/serverless-graalvm-demo/tree/aws-xray-support and it was successful. That doesn't mean it isn't a false positive. I'll test it tomorrow.

Reply:
Sounds good, if removing them causes your test to fail then it's fine to leave as-is

Reply:
Test failed without.